openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -112,10 +126,10 @@ class MeetingActionSet(ActionSet):
 
     model = Meeting()
     create_schema = DefaultSchema(Meeting()).get_create_schema(
-        required_properties=["committee_id", "name"],
-        optional_properties=[*meeting_settings_keys],
+        required_properties=["committee_id", "name", "welcome_title"],
+        optional_properties=meeting_create_payload_fields,
```

Comment:
Why don't you just list the fields directly here? For settings the external array makes sense because of the sheer amount of fields, but as long as they are overseeable I would just list them directly.

Reply:
moved the create_payload fields in the action directly. The array is not that large.